src/app/modules/item/http-services/create-item.service.ts

```diff
@@ -25,8 +25,16 @@ export class CreateItemService {
   }
 
   create(newItem: NewItem): Observable<string> {
+    const body: {[k: string]: any} = {
+      title: newItem.title,
+      type: newItem.type,
+      language_tag: newItem.language_tag,
+    };
+    if ('parent' in newItem) body.parent = newItem.parent;
+    if ('as_root_of_group_id' in newItem) body.as_root_of_group_id = newItem.as_root_of_group_id;
```

Comment:
As discussed this morning and written in the previous comment "Probably you could keep an interface with camelcase attribute". Snakecase should be limited to backend communication. Here, the input is not anymore for backend communication.

Reply:
Fixed